### ISSUE TITLE
Connection error with using Openai API KEY

### ISSUE THREAD

**Issue by TaugenichtsZZY** [FlowiseAI/Flowise#2077]
**Describe the bug**
I followed the "Quick Start" steps on the web page to get to the flowise web site, and used my openai api key to try a simple example, but it always shows a connection error instead of a normal response.

**To Reproduce**

1. Latest NodeJS installed
2. Install and start Flowise
3. Open http://localhost:3000/
4. Try 'Flowise Docs QnA' in Marketplaces

**Screenshots**
![image](https://github.com/FlowiseAI/Flowise/assets/148533515/9a8e2174-d592-43e4-8e04-71756aa84806)

![image](https://github.com/FlowiseAI/Flowise/assets/148533515/235cc0bf-53b1-4bcc-9e1f-de135d8cc351)

**Setup**

-   Installation: npx flowise start
-   Flowise Version: latest
-   OS: Linux
-   Browser: chrome

**Additional context**
This is the first time I've tried a similar app. I'm not sure if it's just the way I'm using it or the way it's installed. I will be using a VPN because of the region, but I get the connection error error whether I use a VPN or not.

**Comment by toi500** [comment]
Did you try to use another loader?

**Comment by toi500** [comment]
Add the pdf loader module, upload a pdf file and see if you got the same error.

**Comment by TaugenichtsZZY** [comment]
> Add the pdf loader node, upload a pdf file and see if you got the same error.

Hello. I did what you said, mimicked the example provided by flowise using the pdf file node and uploaded a pdf file, but it still shows the same error.
![image](https://github.com/FlowiseAI/Flowise/assets/148533515/a697dee9-1780-48c4-b240-1009b64de336)

**Comment by HenryHengZJ** [comment]
can you verify that your openai API call works outside of flowise? - https://platform.openai.com/docs/api-reference/chat/create ?

**Comment by TaugenichtsZZY** [comment]
> can you verify that your openai API call works outside of flowise? - https://platform.openai.com/docs/api-reference/chat/create ?

Yes. I use this openai API key in some browser plugins and in dify (-https://dify.ai)

**Comment by WissemBellara** [comment]
I have the same issue, but I am running flowise ai on docker desktop.

**Comment by liyiyiya** [comment]
> 我有同样的问题，但我在 docker 桌面上运行 flowise ai。

兄弟 你解决了吗
